src/Video/RenderProgram/StaticRenderProgram.hpp

```diff
@@ -17,6 +17,15 @@ namespace Video {
             
             /// Destructor.
             ~StaticRenderProgram();
+            ////Render depth pass
```

Comment:
One slash too much. While at it, for neatness we could add a line break after destructor for some air.